server_development/topics/themes.adoc

```diff
@@ -87,31 +69,12 @@ restarting {project_name}.
 
 .Procedure
 
-ifeval::["{kc_dist}" == "quarkus"]
 . Run Keycloak with the following options:
 +
 [source,bash]
 ----
 bin/kc.[sh|bat] start --spi-theme-static-max-age=-1 --spi-theme-cache-themes=false --spi-theme-cache-templates=false
```

Comment:
If you just run `start-dev` these options are implicitly set as per https://github.com/keycloak/keycloak/blob/e9714c78a6842435b4f9fda624c4306531741ca7/quarkus/runtime/src/main/resources/META-INF/keycloak.conf#L16.

So perhaps we can just have a text stating that:

```
While creating a theme it's a good idea to disable caching as this makes it possible to edit theme resources directly from the `themes` directory without
restarting {project_name}. For that, run the server in development mode after deploying your themes:

[source,bash]
----
kc.sh start-dev
----
```

Reply:
Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.